css/templates/lune/pages/_landing.css

```diff
@@ -1,3 +1,152 @@
 /* //////////////////////////////////////////////////////
    LANDING PAGE
-///////////////////////////////////////////////////////*/
\ No newline at end of file
+///////////////////////////////////////////////////////*/
+
+@import "../page-vars";
+
+:root {
+  --lune-landing-title-offset-horizontal-narrow: -6%;
+  --lune-landing-title-offset-horizontal-tablet: -32%;
+  --lune-landing-title-offset-horizontal-wide: -38%;
+
+  --lune-landing-page-height: 100vh;
+  --lune-landing-page-min-height: 37.5rem; /* 600px */
+}
+
+
+/*******************************************************************************
+ * Tweaking the delay for the entering animations defined in _pages.css
+ ******************************************************************************/
+.lune-landing-title-middle {
+  animation-delay: calc(var(--animation-delay-start) +
+      var(--animation-delay-step));
+}
+
+.lune-landing-title-bottom {
+  animation-delay: calc(var(--animation-delay-start) +
+      2 * var(--animation-delay-step));
+}
+
+.lune-page-landing .lune-component-cta {
+  animation-delay: calc(var(--animation-delay-start) +
+      3 * var(--animation-delay-step));
+}
+
+/*******************************************************************************
+ * Layout & typography
+ ******************************************************************************/
+.lune-page-landing {
+  padding-top: 0;
+  height: auto;
+}
+
+.lune-landing-section {
+  height: var(--lune-landing-page-height);
+  min-height: var(--lune-landing-page-min-height);
+}
+
+.lune-landing-wrapper {
+  top: var(--navbar-height);
+}
+
+.lune-page-landing .lune-component-cta {
+  min-width: 16rem;
+}
+
+/*
+ * When there is enough estate in the viewport, set a fixed height for the page
+ * and fixed position on the lower section, in order to achieve a "reveal" effect.
+ * Threshold is 600px, same as --lune-landing-page-height.
+ */
+@media (min-height: 37.5rem) {
+  .lune-page-landing {
+    height: calc(2 * var(--lune-landing-page-height));
+  }
+
+  .lune-landing-section.lune-theme-dark {
+    position: fixed;
+  }
+
+  /*
+   * Only apply this enhancement if clip-path is supported
+   * and the viewport is high enough.
+   */
+  @supports (clip-path: inset(1px 0)) {
+    .lune-landing-section.lune-theme-light {
+      clip-path: url(#svgPath);
+      /*
+       * We need the height to be a bit more than its normal value (hence the 1.02 factor),
+       * as the clip-path mask can't clip the entire surface.
+       */
+      height: calc(var(--lune-landing-page-height) * 1.02);
+    }
+
+    .lune-landing-wrapper {
+      position: fixed;
+    }
+  }
+}
+
+/*
+ * Because amp-img can't be resized in a responsive way through changing its height,
```

Comment:
I tried using `heights` (and also `layout="fill"`), but I couldn't really achieve the responsive behavior I need, unless I used the `object-fit` CSS property.
Although the [docs](https://ampbyexample.com/advanced/how_to_support_images_with_unknown_dimensions/#object-fit-to-the-rescue) encourage the use of `object-fit`, we can't really use it as its browser support isn't good enough (no IE, no edge).

I believe that the current solution is a good one, as it fulfills completely the design requirements and works across all major browsers